pmlc/dialect/pxa/transforms/heatmap.h

```diff
@@ -0,0 +1,14 @@
+// Copyright 2018, Intel Corporation
```

Comment:
Can we move these files over to `pmlc/target/x86` too?

Reply:
Will do.